aea/helpers/dialogue/base.py

```diff
@@ -210,43 +210,3 @@ def _next_dialogue_id(self) -> int:
         """
         self._dialogue_id += 1
         return self._dialogue_id
-
-    def create_self_initiated(self, dialogue_opponent_pbk: str, dialogue_starter_pbk: str) -> Dialogue:
```

Comment:
These are too specific implementations, hence they are removed from the base class.